framework/util/driver_info.cpp

```diff
@@ -172,6 +281,48 @@ bool GetDriverInfo(std::string& driver_info, format::ApiFamilyId api_family)
     return driver_info_available;
 }
 
+bool RegistryDxDriverVersion(std::string& driver_info, const std::vector<LUID>& adapter_luids)
+{
+    bool driver_info_found = false;
+
+    HKEY    dx_key_handle      = nullptr;
+    DWORD   num_of_adapters    = 0;
+    DWORD   sub_key_max_length = 0;
+    LSTATUS return_code        = GetRegSubkeys(dx_key_handle, num_of_adapters, sub_key_max_length);
+    if (return_code == ERROR_SUCCESS)
+    {
+        return_code = GetRegData(dx_key_handle, num_of_adapters, sub_key_max_length, driver_info, adapter_luids);
+        if (return_code == ERROR_SUCCESS)
+        {
+            driver_info_found = true;
+        }
+    }
+
+    return driver_info_found;
+}
+
+#endif
+
+std::string ConvertDWORDtoVersionNumber(uint64_t dword)
```

Comment:
I think a DWORD is still 32 bits on 64 bit Windows. Does this function and the argument to it need tweaked names?
https://learn.microsoft.com/en-gb/windows/win32/winprog/windows-data-types
https://learn.microsoft.com/en-us/windows/win32/winprog64/the-new-data-types
https://godbolt.org/z/4sbjG85nb

Reply:
fixed